src/renderer/setupPasswordOptional.html

```diff
@@ -0,0 +1,304 @@
+<!DOCTYPE html>
+<html>
+  <head>
+    <meta charset="utf-8" />
+    <link rel="stylesheet" type="text/css" href="index.css" />
+  </head>
+  <style>
+    .pw-input {
+      height: 30px;
+      width: 200px;
+      border-radius: 5px;
+      font-size: 16px;
+    }
+
+    button {
+      height: 26px;
+      min-width: 80px;
+      cursor: pointer;
+    }
+
+    .warning {
+      color: rgb(193, 0, 0);
+      font-size: 14px;
+      min-height: 16px;
+    }
+
+    .red {
+      color: rgb(193, 0, 0);
+    }
+
+    .previous-step-btn {
+      position: fixed;
+      top: 6px;
+      left: 6px;
+    }
```

Comment:
This code also apprears to be duplicated, perhaps separating the common styles into a separate css file would be better

Reply:
Okay, moved to the common css file :+1: